packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -67,16 +68,29 @@ export type TabsProps = {
    * Identifier for the selected tab.
    */
   value?: string;
+  /**
+   * Callback fired when the active tab is changed.
+   */
+  onChange?: (event: SyntheticEvent, value: string) => void;
```

Comment:
This should be `MouseEventHandler<HTMLDivElement>`.

Because MUI controls the type and HTML element, we can make it `MuiTabListProps["onChange"]` instead. It's weird to set it to their function because we're wrapping it, but like I said, they're in control, not us.